### PR TITLE
feat: add method update

### PR DESCRIPTION
**Pull request by SocolaDaiCa**

### CODE REVIEW

File: src/Settings.php [hunk 1]
```diff
@@ -183,6 +183,11 @@ public function save(): self
         return $this;
     }
 
+    public function update($properties): self
```
Comment (rcerljenko) [inline src/Settings.php:186]: you should probably type `$properties` as `array`

**Review by rcerljenko — CHANGES_REQUESTED** [review]

File: src/Settings.php (hunk 1 above)
Comment (rubenvanassche) [reply src/Settings.php:186]: Indeed

**Comment by rubenvanassche** [comment]
Could you also add a test which tests this feature?

**Comment by rubenvanassche** [comment]
Closed due inactivity.

**Pull request closed without merging**